lib/scala/pkg/src/main/resources/default/src/Main.enso

```diff
@@ -2,6 +2,7 @@ from Standard.Base import all
 from Standard.Table import all
 from Standard.Database import all
 from Standard.AWS import all
+from Standard.Google_Api import all
```

Comment:
Shouldn't the IDE be fixed to _not require_ imports of all libraries we have and rather add such imports on demand when a component from a particular library is used? Then we could remove `AWS` and possibly also `Table` and `Database`.

Is this issue tracked somewhere?

Reply:
https://github.com/enso-org/enso/issues/9413